src/components/RemindersManager.test.tsx

```diff
@@ -76,6 +76,10 @@ describe('Reminders manager', () => {
 		const mocks = [mockFindTasks({ status: Status.Open }, [task])];
 		setup(<RemindersManager />, { mocks, initialRouterEntries: [`/${TASKS_ROUTE}`] });
 		await waitForModalToOpen();
+		act(() => {
+			// run modal timers
+			jest.runOnlyPendingTimers();
+		});
```

Comment:
Advance the timers inside the waitForModalToOpen funcion (use the jest.advanceTimersByTime)